.github/workflows/reusable-build.yaml

```diff
@@ -14,6 +14,8 @@ jobs:
       run: pbuilder-dist stable arm64 create
     - name: Pbuilder arm64 update
       run: pbuilder-dist stable arm64 update
+    - name: Install bazel
+      run: cd tools/cuttlefish-host-image-installer && ./utils/install_bazel_package.sh
```

Comment:
Use `https://github.com/google/android-cuttlefish/blob/main/tools/buildutils/installbazel.sh` instead.

Reply:
Since it is cross-platform compilation (we don't have ARM64 github runner yet), we can't directly execute installbazel.sh or install_bazel_package.sh. Instead, I have to hardcode function install_bazel_aarch64() part of codes into build_cf_packages.sh.
When ARM64 github runner is available, I will change it to execute installbazel.sh directly.